squad-server/plugins/auto-tk-warn.js

```diff
@@ -11,10 +11,15 @@ export default class AutoTKWarn extends BasePlugin {
 
   static get optionsSpecification() {
     return {
-      message: {
+      attackerMessage: {
         required: false,
-        description: 'The message to warn players with.',
+        description: 'The message to warn attacking players with.',
         default: 'Please apologise for ALL TKs in ALL chat!'
+      },
+      victimMessage: {
+        required: false,
+        description: 'The message that will be sent to the victim.',
+        default: 'You were killed by your own team.'
```

Comment:
One last thing, I think we should default this to `null` so it's not enabled by default.

Reply:
yeah that was the original reason for the separate flag I'll set it to null and comment out the suggested message